arch/M680X/M680XDisassembler.c

```diff
@@ -8,6 +8,9 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <string.h>
+#ifdef CAPSTONE_DEBUG
```

Comment:
We could move this into cs_priv.h so that other files don't need to manually include.

Reply:
Done